### PR TITLE
Add ScaleSelect to print form

### PR DESCRIPTION
**Pull request by annarieger**
* Introduce `ScaleSelect` component to print form which allows to choose print scale via dropdown
* Allow to pass an optional parameter `customPrintScales` to the `PrintForm` 
* Remove hardcoded values for initial DPI and output formats, use dynamic values set on mapfish print provider instead
* Get rid of some `@ts-ignore` annotations

Please review @terrestris/devs

### CODE REVIEW

File: src/components/PrintForm/ScaleSelect/index.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+export const ScaleSelect: React.FC<ScaleSelectProps> = ({
+  printManager,
+  placeholder,
+  value,
+  ...restProps
+}): JSX.Element => {
+
+  const map = useMap();
+
+  const [scale, setScale] = useState<any>(value);
```
Comment (dnlkoch) [inline src/components/PrintForm/ScaleSelect/index.tsx:32]: Any chance to get rid of the `any` type?

File: src/components/PrintForm/ScaleSelect/index.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+  useEffect(() => {
+    if (printManager) {
+      printManager.setScale(scale);
+    }
+  }, [printManager, scale]);
+
+  useEffect(() => {
+
```
Comment (dnlkoch) [inline src/components/PrintForm/ScaleSelect/index.tsx:41]:
```suggestion
```

**Review by dnlkoch — APPROVED** [review]

File: src/components/PrintForm/ScaleSelect/index.tsx (hunk 1 above)
Comment (annarieger) [reply src/components/PrintForm/ScaleSelect/index.tsx]: done with 88360d8

**Pull request merged**